assets/lib/font-awesome/css/fontawesome.min.css

```diff
@@ -1,5 +1,6 @@
 /*!
```

Comment:
What is the difference between the files with the `-` and without?
Why do we need both `fontawesome.css` and `font-awesome.css`

Reply:
I can't find the file font-awesome.css, maybe I already deleted it.